### DISCUSSION TITLE
Study Group Questions - Place all study group related questions in this ticket.

### DISCUSSION THREAD

**Discussion by drteresavasquez** [nss-evening-cohort-14/evening-client-side#48] [Discuss a Topic]
General questions about study group and assignments. Please place all study group related questions in this ticket.

**Comment by drteresavasquez** [comment]
@nss-evening-cohort-14/students I know you have questions over the study group assignments this week. Post them here and eihter another teammate can jump in to help or we can help as well. Getting you prepared for planning and teamwork is an essential part of dev life. We want to expose you to errthang early so when you need the skills, you will have prepped for them.

**Comment by tsekeres** [comment]
Some questions I asked and got clarity from Dr. T. on:
1. Are these group study tasks designed to be completed in one three hour session? 
         These tasks may very well take longer than three hours. You usually have a week to do them for a reason. But you might get, as a group, comfortable enough to not have to meet again together. Then you can work on the assignment with support over slack and such.

2. Are these study group assignments really intense or am I far more clueless than I thought? 
        They are designed to stretch you. But have some familiar steps so that you can grow. But you gotta work for it and learn to search and conquer as a team.

3. Is this group work or individual work?
        I figured out that this is the wrong way to think about this process. It is not an assignment to check off/turn in/get a grade for. This is an activity about a skill or process that you should be familiar with before moving on. So "finishing" the assignment looks like the whole group working together and individually becoming comfortable and knowledgeable about the topic at hand. Comfortable enough to move your card to the Done column. So if it is a group type task (like setting up a group project on github/figma), you should all do it individually to become familiar and at the same time to support each other through it.

TLDR: 1. Tasks take a week or less. Maybe three hours. Maybe not. 2. Study group tasks are hard but you can do it! 3. Individual skill work done together for support!

**Reply by drteresavasquez** [reply]
Boom there it is!

![image](https://user-images.githubusercontent.com/29741570/105428550-31880b00-5c15-11eb-93af-e4c52d85ed2b.gif)

**Comment by hmparsons-sde** [comment]
So sorry if I missed this, but should we expect to complete one task per week? In other words, do we have the next 3 weeks of group cut out for us? Thanks in advance!

**Reply by tsekeres** [reply]
No, I believe each of the assignments that dropped this week have a due date of  1/26. You have to watch the due dates.

Side note. There is no "project" to do with this planning assignment yet. The assignment is to just LEARN HOW to set it up a shared repo and project with figma and project board.

**Reply by hmparsons-sde** [reply]
thank you! I appreciate the quick response :)

**Comment by Gabrielle-Tobermann** [comment]
I'm trying to create the wireframe for the Figma assignment. I need to upgrade to the education version to access the styles and components of the Mobile Wireframe UI kit. But when trying to sign up for the Education Plan, NSS is not listed... Did anybody get around this?

**Reply by drteresavasquez** [reply]
The free version should work. But you can select "school not listed" and complete the form.

**Comment by hmparsons-sde** [comment]
Hi all,

I am having a hard time deploying my site to Netlify. This error keeps returning: 

"3:29:27 PM: Failing build: Failed to parse configuration
3:29:27 PM: Failed during stage 'Reading and parsing configuration files':
Base directory does not exist: /opt/build/repo/lib/app-root-path.js"

Things I've tried:

1. Scoured the Netlify community pages - from there I installed a node_modules (app-root-path module) folder, because Netlify looks for package.json. 
2. StackOverflow offered little help, referring mainly to React and Gatsby apps.
3. I looked at the How to Geek blog (I know, I know), which offered advice about local base directories, but that did not work.

What am I missing?

![screemshot](https://user-images.githubusercontent.com/67122062/106331882-5569d300-624b-11eb-9b6c-868c3d4782e6.png)

**Reply by hmparsons-sde** [reply]
https://github.com/hmparsons-sde/pet-adoption/tree/main/node_modules/app-root-path

**Reply by drteresavasquez** [reply]
We will go over deployment tomorrow morning. It could be that the publish directory is not set properly. Can you screenshot your build settings? Should look like this:

![image](https://user-images.githubusercontent.com/29741570/106332308-2738c300-624c-11eb-97a2-d2e7e0d8cf76.png)

**Reply by drteresavasquez** [reply]
Your publish directory will have a different input but I just want to see yours.

**Reply by hmparsons-sde** [reply]
Oh! If we're talking about it tomorrow, you will probably answer my question :)

![2021-01-29 (1)](https://user-images.githubusercontent.com/67122062/106332432-6d8e2200-624c-11eb-9180-099c96e0db8c.png)

**Reply by drteresavasquez** [reply]
So, make sure that all your code is on your master/main branch and under your Build and Deploy, you should have this:

NOTE that it is mostly empty
<img width="1378" alt="Screen Shot 2021-01-29 at 4 20 54 PM" src="https://user-images.githubusercontent.com/29741570/106333379-26089580-624e-11eb-9a6a-e6bab43018a6.png">

**Reply by hmparsons-sde** [reply]
aha! it deployed my index.html, but not main.js. I'll wait to hear more tomorrow. Thank you for your time!
